plugins/dell-kestrel/fu-dell-kestrel-plugin.c

```diff
@@ -437,6 +437,9 @@ fu_dell_kestrel_plugin_constructed(GObject *obj)
 	fu_plugin_add_device_gtype(plugin, FU_TYPE_DELL_KESTREL_DPMUX);
 	fu_plugin_add_device_gtype(plugin, FU_TYPE_DELL_KESTREL_WTPD);
 	fu_plugin_add_device_gtype(plugin, FU_TYPE_DELL_KESTREL_ILAN);
+	fu_plugin_add_device_gtype(plugin, FU_TYPE_DELL_KESTREL_RMM);	 /* coverage */
+	fu_plugin_add_device_gtype(plugin, FU_TYPE_DELL_KESTREL_EC);	 /* coverage */
+	fu_plugin_add_device_gtype(plugin, FU_TYPE_DELL_KESTREL_RTSHUB); /* coverage */
```

Comment:
The three devices have their own device id to be added to the plugin through backend_device_added, e.g.:
https://github.com/fwupd/fwupd/blob/wip/cragw/kestrel/plugins/dell-kestrel/fu-dell-kestrel-plugin.c#L75
https://github.com/fwupd/fwupd/blob/wip/cragw/kestrel/plugins/dell-kestrel/fu-dell-kestrel-plugin.c#L100
https://github.com/fwupd/fwupd/blob/wip/cragw/kestrel/plugins/dell-kestrel/fu-dell-kestrel-plugin.c#L181

Should we claim gtype for them in this case?

Reply:
This is just adding them as "possible" gtypes so that the coverage checker can instantiate them.